src/db/schema.ts

```diff
@@ -1,11 +1,230 @@
-import { integer, pgTable, varchar } from "drizzle-orm/pg-core";
+import {
+  boolean,
+  index,
+  integer,
+  pgEnum,
+  pgTable,
+  real,
+  text,
+  timestamp,
+  uuid,
+  varchar,
+  vector,
+} from "drizzle-orm/pg-core";
+
+// ------------------------------------------
+// ENUMS
+// ------------------------------------------
+
+export const userRoleEnum = pgEnum("user_role", ["member", "creator", "admin"]);
+
+export const requestStatusEnum = pgEnum("request_status", ["pending", "approved", "rejected"]);
+
+export const eventVisibilityEnum = pgEnum("event_visibility", ["public", "unlisted"]);
+
+export const photoStatusEnum = pgEnum("photo_status", [
+  "pending",
+  "processing",
+  "ready",
+  "failed",
+  "hidden",
+  "deleted",
+]);
+
+export const claimStatusEnum = pgEnum("claim_status", ["pending", "approved", "rejected"]);
+
+// ------------------------------------------
+// AUTHENTICATION (Better-Auth)
+// ------------------------------------------
 
 export const usersTable = pgTable("users", {
```

Comment:
This is a refactor. I've consulted with a few people, and decided that we will use singular table name (`users` → `user`, `photos` → `photo`). I won't comment one by one on every instances, but please change the name of every table to singular (just like the `session`, `account`, and `verification` tables, actually). Sorry about that.

Furthermore, drop the `Table` suffix for the model names (e.g., `usersTable` → `user`, `photosTable` → `photo`, `photoEmbeddingsTable` → `photoEmbedding`).

Lastly, adjust the dbml schema based on these changes.

```suggestion
export const user = pgTable("user", {
```